Doc/library/math.rst

```diff
@@ -166,6 +166,14 @@ Number-theoretic and representation functions
    Return ``True`` if *x* is a NaN (not a number), and ``False`` otherwise.
 
 
+.. function:: isqrt(n)
+
+   Return the "integer square root" of the integer *n*. This is the largest
+   integer *s* not exceeding the exact square root of *n*.
```

Comment:
From comments on the issue: add the recipe `1 + isqrt(n - 1)` for computing the *ceiling* of the square root of a positive integer.

Reply:
Done.